UPGRADING.md

```diff
@@ -2,6 +2,17 @@
 
 This guide provides instructions for upgrading to specific versions of Replicated Security.
 
+## [v4.1.x](https://github.com/cosmos/interchain-security/releases/tag/v4.1.0-rc0) and [v4.1.x-lsm](https://github.com/cosmos/interchain-security/releases/tag/v4.1.0-lsm-rc0)
+
+### Provider
+
+Upgrading a provider from `v4.0.0` to `v4.1.0` or `v4.1.0-lsm` requires state migrations, see relevant pull request [here](https://github.com/cosmos/interchain-security/pull/1757),
```

Comment:
This is not correct. v4.0.0 and v4.1.1 have the same consensus version if I'm reading the code correctly.

If you upgrade from v4.0.0 to v4.1.0 no state upgrade handlers will execute because the consensus version is the same

Reply:
v4.1.0 consensus version:
* https://github.com/cosmos/interchain-security/blob/release/v4.1.x/x/ccv/provider/module.go#L137

v4.0.0 consensus version:
* https://github.com/cosmos/interchain-security/blob/release/v4.0.x/x/ccv/provider/module.go#L137

Reply:
I'll add a fix and update this doc when the fix is in